util/OptionsDB.cpp

```diff
@@ -137,8 +137,7 @@ OptionsDB::OptionsDB() {
     s_options_db = this;
 }
 
-void OptionsDB::Commit()
-{
+void OptionsDB::Commit() {
```

Comment:
> At the time, the Wiki stated that I could do what I preferred, if it didn't conflict with the style of the existing code (which had opening brackets done in more than one way).

There are several problems with this in general.

@geoffthemedio is right with his conclusion, topics and *grooming/housekeeping/whatever you want to call it* should be IMO separated at least by commits or even better left out completely from PRs. Why do I ask for this:

*  When looking back into the source code history via git to understand the implementation idea or asking the responsible developer why some code was done this way I don't want to step to some large number of unrelated grooming commits.
* A commit should be atomic to allow the most easy revert when needed. This is easier when a commit only holds a logical connected part. When mixing in changes that are unrelated (grooming or a different topic) this is way harder to to because the possibility for revert conflicts is higher.

Why are there are inconsistent styles: Well, it's old code and there was and is no technical enforcement of style that would prevent committing to the code.  Also there is no consensus what well-formed code style should look like and how existing inconsistencies to that theoretical existing well-formed code style should be purged.  So the minimal agreement currently is:

* Use the style used in the same file.
* Don't add pointless grooming commits.
* Prefer fixing inconsistencies when touching a code line for other reasons.